aries-site/src/examples/components/tooltip/ToolTipSizeExample.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Box, Button, Tip, Text } from 'grommet';
+
+export const ToolTipSizeExample = () => (
```

Comment:
To align with language, should we update all of these files/folders to `tip` instead of `tooltip`?